conformance/utils/suite/suite.go

```diff
@@ -89,7 +89,7 @@ func New(s Options) *ConformanceTestSuite {
 	}
 
 	switch {
-	case s.EnableAllSupportedFeatures == true:
```

Comment:
Seems likely to be an IDE edit? It's fine in this case, but in general it's better to put this sort of change in a separate PR.

Reply:
Yep: https://github.com/kubernetes-sigs/gateway-api/pull/2346